### PR TITLE
Onboarding new design - Wink Hub

### PR DESCRIPTION
**Pull request by sanjaiganesh**
Defines Wink Hub translator, in the new 'Hub as a translator' model. Expecting more changes, as there are still some gaps/open questions around accounts, multiple hubs etc..

### CODE REVIEW

**Comment by sanjaiganesh** [comment]
@tjaffri @jasongin @arjun-msft @jeanpa Please take a look.

File: org.OpenT2T.Onboarding.WinkHub/v2/winkonboarding.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+// module exports, implementing the schema
+module.exports = {
+
+    //
+    // question: Do we need to remove validate function?
+    // This is tied with 'inquirer' syntax, which helps validate at commandline as well.
+    //
+    onboardingFlow : [
+        {
+            action: 'get-user-input',
```
Comment (arjun-msft) [inline org.OpenT2T.Onboarding.WinkHub/v2/winkonboarding.js]: I see two different data sets in here, one set which the *developer* provides (client_id, client_secret) and the other which the *end user* provides (username, password).  They have to be separately acquired.  The developer ones will likely be passed in like the nest sample and the end user ones like the osram ones does.  So I expect to see two items in the onboarding flow array, 'get-use-input' and 'get-developer-input'

Comment (sanjaiganesh) [inline org.OpenT2T.Onboarding.WinkHub/v2/winkonboarding.js]: Sure. Should be good to have 'separation of concerns'

---
In reply to: [74331940](https://github.com/openT2T/onboarding/pull/10#discussion_r74331940) [](ancestors = 74331940)

File: org.OpenT2T.Onboarding.Hub/onboarding.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Onboarding Scenarios
+
+Onboarding allows dvelopers to add devices to the opent2t system allowing opent2t apps and services to talk to them.
```
Comment (begillil) [inline org.OpenT2T.Onboarding.Hub/onboarding.md]: Typo: dvelopers -> developers

File: org.OpenT2T.Onboarding.Hub/opent2tonboarding.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    {
+        this.accessToken = accessToken;
+        this.refreshToken = refreshToken;
+        this.tokenType = tokenType;
+        this.scopes = scopes;
+    }
+}
+
+// Consumers must be aware and handle the case that not all properties are available all the time & for all devices. (check for 'undefined')
+class deviceInfo{
```
Comment (busadave13) [inline org.OpenT2T.Onboarding.Hub/opent2tonboarding.js]: Can we add a device type/category identifier (e.g. thermostat, light) that can be used to associate the device with translator

**Comment by arjun-msft** [comment]
This work has been done, this pr is out of date.

**Pull request closed without merging**